tensorflow/contrib/timeseries/python/timeseries/estimators.py

```diff
@@ -22,6 +22,7 @@
 from tensorflow.contrib.timeseries.python.timeseries import feature_keys
 from tensorflow.contrib.timeseries.python.timeseries import math_utils
 from tensorflow.contrib.timeseries.python.timeseries import model_utils
```

Comment:
The model_utils import should now be unused here, right? There is a model_utils.state_to_dictionary call, but that got moved and looks like it needs updating. You could even move the serving input receiver stuff to the head lib if you wanted.